kgforge/specializations/stores/nexus/service.py

```diff
@@ -449,6 +445,15 @@ async def dispatch_action():
 
         return asyncio.run(dispatch_action())
 
+    def _prepare_tag(self, resource, tag) -> Tuple[str, str, str, str]:
+        schema_id = resource._store_metadata._constrainedBy
+        schema_id = "_" if schema_id == self.UNCONSTRAINED_SCHEMA or schema_id is None else schema_id
+        url = "/".join((self.url_resources, quote_plus(schema_id), quote_plus(resource.id), "tags"))
+        rev = resource._store_metadata._rev
+        params = {"rev":rev}
+        data = {"tag": tag, "rev": rev}
+        return url, data, params, schema_id
```

Comment:
why returning the schema_id if it is never used?

Reply:
schema_id is removed